.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 rvm:
   - 2.4.6
 
+before_script: export SCHEMA_VERSION=0.7
```

Comment:
Let's define the specific version we are validating for here and not mess with the file structure. @makeworld-the-better-one in your proposal the current version is no in the file path, we don't want that.